frontend/app/src/components/settings/general/rpc/EvmRpcNodeManager.vue

```diff
@@ -130,8 +120,8 @@ function showDeleteConfirmation(item: EvmRpcNode) {
   );
 }
 
-onUnmounted(() => {
-  disposeEvmRpcNodeComposables();
```

Comment:
this `disposeEvmRpcNodeComposables` function can be removed in `settings/general/rpc/form.ts` now I think since no other usage for this?

Reply:
I removed the whole `settings/genera/rpc/form.ts` since I changed the approach